packages/cli/src/commands/remove-plugin/removePlugin.test.ts

```diff
@@ -68,8 +67,8 @@ const createTestPluginFile = async (
     .split('-')
     .map(name => capitalize(name))
     .join('');
-  const exportStatement = `export { default as ${pluginNameCapitalized}} from @backstage/plugin-${testPluginName}`;
-  addExportStatement(testFilePath, exportStatement);
+  const exportStatement = `export { plugin as ${pluginNameCapitalized}} from @backstage/plugin-${testPluginName}`;
+  await addExportStatement(testFilePath, exportStatement);
```

Comment:
This is what expect will fix the build failures, the rest is just cosmetic changes